tools/sunlight.go

```diff
@@ -263,6 +306,17 @@ func main() {
 				fmt.Fprintf(os.Stderr, "Couldn't write json: %s\n", err)
 				os.Exit(1)
 			}
+
+			exampleMapLock.Lock()
+			if exampleMap[cert.Issuer.CommonName] == nil {
+				exampleMap[cert.Issuer.CommonName] = make(map[string]*x509.Certificate)
+			}
+			for violation, isViolation := range summary.Violations {
+				if isViolation {
+					exampleMap[cert.Issuer.CommonName][violation] = cert
```

Comment:
Unless I'm not understanding what I wrote, this line will overwrite any previous example issued by that issuer for that violation, right?